src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (element) => {
+  element.preventDefault();
```

Comment:
Use 'event' instead of 'element' as parameter name to better reflect its purpose